docs/reference/commandline/tag.md

```diff
@@ -35,7 +35,7 @@ components may contain lowercase letters, digits and separators. A separator
 is defined as a period, one or two underscores, or one or more dashes. A name
 component may not start or end with a separator.
 
-A tag name must be valid ASCII and may contain lowercase and uppercase letters,
+A tag name must be valid ASCII and may contain lowercase letters,
```

Comment:
Repository names can't have uppercase characters. There may be confusion between the repository and the tag.